easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-5.2.0.eb

```diff
@@ -0,0 +1,28 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'icc'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_%(version)s.tgz']
+
+checksums = ['f57a892fb494db3c80f20a88aa3e901f']
+
+gcc = 'GCC'
+gccver = '5.2.0'
+versionsuffix = '-%s-%s' % (gcc, gccver)
+
+dependencies = [(gcc, gccver)]
+
+dontcreateinstalldir = 'True'
+
+# license file
+import os
+license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+license_file = os.path.join(os.getenv('HPC_BUILDENV_ROOTPATH'), "licenses", "intel", "USE_SERVER.lic")
```

Comment:
@DirkdeDraak: don't include site-specific things like this in easyconfigs you contribute

In fact, there's no need to modify the easyconfig at all to make sure your easyconfig file is picked up.

If you define `$INTEL_LICENSE_FILE` in your environment, the `IntelBase` easyblock will pick it up and use it (which gets preference over the `license_file` value defined here).